keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt

```diff
@@ -153,32 +153,32 @@ class ImageService(
     }
   }
 
-  suspend fun findBaseAmiVersion(baseImageName: String): String {
+  suspend fun findBaseAmiName(baseImageName: String): String {
     return cloudDriverService.namedImages(DEFAULT_SERVICE_ACCOUNT, baseImageName, "test")
       .lastOrNull()
       ?.let { namedImage ->
-        findBaseAmiVersion(namedImage)
+        findBaseAmiName(namedImage)
       } ?: throw BaseAmiNotFound(baseImageName)
   }
 
-  suspend fun findBaseAmiVersion(baseImageId: String, region: String): String {
+  suspend fun findBaseAmiName(baseImageId: String, region: String): String {
     return cloudDriverService.getImage("test", region, baseImageId)
       .lastOrNull()
       ?.let { namedImage ->
-        findBaseAmiVersion(namedImage)
+        findBaseAmiName(namedImage)
       } ?: throw BaseAmiNotFound(baseImageId)
   }
 
   /**
    * Extracts the base ami version from the tags of a named image
    */
-  private fun findBaseAmiVersion(image: NamedImage): String? =
+  private fun findBaseAmiName(image: NamedImage): String? =
     image
       .tagsByImageId
       .values
       .filterNotNull()
-      .find { it.containsKey("base_ami_version") }
-      ?.getValue("base_ami_version")
+      .find { it.containsKey("base_ami_name") }
+      ?.getValue("base_ami_name")
```

Comment:
This reflects the change @ajordens made in CloudDriver so that this new `base_ami_name` tag will be present